extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { memo } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { STATE_ERROR } from '../constants';
+
+const TrackError = memo( ( { link } ) => (
+	<div className="podcast-player__episode-error">
+		{ __( 'Episode unavailable', 'jetpack' ) }{ ' ' }
+		{ link && (
+			<span>
+				{ '(' }
+				<a href={ link } rel="noopener noreferrer nofollow" target="_blank">
+					{ __( 'Open in a new tab', 'jetpack' ) }
+				</a>
+				{ ')' }
+			</span>
+		) }
+	</div>
+) );
+
+const Track = memo( ( { track, isActive, isError, selectTrack, index } ) => {
+	return (
+		<li
+			className={ classnames( 'podcast-player__episode', {
+				'is-active': isActive,
+			} ) }
+		>
+			<a
```

Comment:
If we're generating this via JS now (rather than progressively enhancing it off a list of links), is it OK to modify this HTML structure to not use the `<a>` element? We could utilize a `<button>` or possibly styled `<input type="radio" />` depending on what would be the best implementation. That would prevent us from needing to implement our own `spacebar` listener. If that's the case, we can make a follow-up PR for this.

Reply:
Changing the element will prevent us from having the cmd+click functionality to open an episode. `<a>` is the only one that can provide this

Reply:
Sounds good, as I had already thought out a way to introduce appropriate `aria` rules using an `<a>` 😅